rules/windows/process_creation/proc_creation_win_registry_enumeration_for_credentials_cli.yml

```
@@ -21,24 +22,30 @@ logsource:
 detection:
     selection:
         CommandLine|contains: # Add more paths as they are discovered
-            - '\Software\SimonTatham\PuTTY\Sessions'
             - '\Software\\SimonTatham\PuTTY\SshHostKeys\'
-            - '\Software\Mobatek\MobaXterm\'
-            - '\Software\WOW6432Node\Radmin\v3.0\Server\Parameters\Radmin'
-            - '\Software\Aerofox\FoxmailPreview'
             - '\Software\Aerofox\Foxmail\V3.1'
+            - '\Software\Aerofox\FoxmailPreview'
+            - '\Software\DownloadManager\Passwords'
+            - '\Software\FTPWare\COREFTP\Sites'
             - '\Software\IncrediMail\Identities'
-            - '\Software\Qualcomm\Eudora\CommandLine'
-            - '\Software\RimArts\B2\Settings'
-            - '\Software\OpenVPN-GUI\configs'
             - '\Software\Martin Prikryl\WinSCP 2\Sessions'
-            - '\Software\FTPWare\COREFTP\Sites'
-            - '\Software\DownloadManager\Passwords'
+            - '\Software\Mobatek\MobaXterm\'
             - '\Software\OpenSSH\Agent\Keys'
-            - '\Software\TightVNC\Server'
+            - '\Software\OpenVPN-GUI\configs'
             - '\Software\ORL\WinVNC3\Password'
+            - '\Software\Qualcomm\Eudora\CommandLine'
             - '\Software\RealVNC\WinVNC4'
-    condition: selection
+            - '\Software\RimArts\B2\Settings'
+            - '\Software\SimonTatham\PuTTY\Sessions'
+            - '\Software\Sota\FFFTP'
+            - '\Software\TightVNC\Server'
+            - '\Software\WOW6432Node\Radmin\v3.0\Server\Parameters\Radmin'
+    filter:
+        Image|endswith: 'reg.exe'
+        CommandLine|contains:
+            - 'export'
+            - 'save'
+    condition: selection and not filter
```

```suggestion
    filter_main_other_rule:  # matched by cc1abf27-78a3-4ac5-a51c-f3070b1d8e40
        Image|endswith: 'reg.exe'
        CommandLine|contains:
            - 'export'
            - 'save'
    condition: selection and not 1 of filter_main_*
```
You do this bc of the new rule for higher level alerting. So please add relations to both rules. (In the future the high-level rule probably should be a correlation rule)